projects/core/components/dialog/dialogs.component.ts

```diff
@@ -33,6 +33,9 @@ import {TUI_DIALOGS} from './dialog.tokens';
 })
 export class TuiDialogs {
     protected readonly el = tuiInjectElement();
-    protected readonly dialogs$: Observable<ReadonlyArray<TuiPopover<any, any>>> =
-        inject(TUI_DIALOGS);
+    protected readonly dialogs = toSignal<ReadonlyArray<TuiPopover<any, any>>>(
```

Comment:
it looks like the overlay is now blinking

https://github.com/user-attachments/assets/fb2a8eba-4769-4c7e-bfdf-34740f51fec1

Reply:
fixed